aeron-driver/src/main/c/aeron_retransmit_handler.c

```diff
@@ -31,13 +31,22 @@ int aeron_retransmit_handler_init(
     aeron_retransmit_handler_t *handler,
     int64_t *invalid_packets_counter,
     uint64_t delay_timeout_ns,
-    uint64_t linger_timeout_ns)
+    uint64_t linger_timeout_ns,
+    bool is_multicast)
 {
     handler->invalid_packets_counter = invalid_packets_counter;
     handler->delay_timeout_ns = delay_timeout_ns;
     handler->linger_timeout_ns = linger_timeout_ns;
+    handler->is_multicast = is_multicast;
```

Comment:
Need to take into account MDC & `group=true` (see comment on Java implementation).